src/warp/yul/BuiltinHandler.py

```diff
@@ -730,8 +730,15 @@ class LinkerSymbol(NotImplementedStarkNet):
     pass
 
 
-class LoadImmutable(NotImplementedStarkNet):
-    pass
+class LoadImmutable(DynamicHandler):
+    def __init__(self, cairo_functions: CairoFunctions, imm_dict: dict[str, int]):
+        super().__init__(cairo_functions.load_immutable_function)
+        self.imm_dict = imm_dict
+
+    def get_function_call(self, function_args: Sequence[str]) -> str:
+        (arg,) = function_args
+        id = self.imm_dict.get(arg)
```

Comment:
This causes a typing error. `imm_dict.get` might return `none` if there's no value associated with `key`. In that case, we should throw an exception to the user